### ISSUE TITLE
Sound glitches when EDDI use Cortana (Microsoft Eva) voice

### ISSUE THREAD

**Issue by Ipslor** [EDCD/EDDI#445]
## EDDI version in which issue found
3.0.0-rc1

## Steps to reproduce
1. Lauch EDDI
2. Go to TTS tab
3. Set 'Voice' setting to 'Microsoft Eva Mobile' or 'Microsoft Eva Desktop'
4. Launch E:D
5. Perform high wake (jump between stars) or drop to normal space from cruise

## Expected
EDDI says like 'Telemetry obtained' or 'Dropped to normal space'

## Observed
Weird glitching sound like scratches

## Investigation
Other voices work pretty well. Issue present at all 3.x versions of EDDI
OS: Windows 10 Pro

**Comment by HighwaymanEd** [comment]
Exact same issue here

**Comment by Tkael** [comment]
Each voice on your system works like a different program and has quirks that cause it to adhere more or less to published standards. Companies that sell voices typically document the area that comply or don't comply to the published standards, but we don't have that for Microsoft Eva. 
Without any published documentation, and given that Microsoft Eva isn't a voice that Microsoft supports outside of their Cortana app, we don't think we can offer support for it.

**Comment by Keeper1st** [comment]
Old issue, but I looked it up now because I just discovered while messing about that the distortions are caused by the Pause value before speech begins. The longer the Pause value, the more distortion. It can be rectified simply by adding a "." at the end of the value, e.g. "{Pause(3000)}." This makes it basically "end the sentence", and the new sentence begins without the distortion. Whether there are any other issues with the voice, I couldn't say yet. I'll have to modify my Pause values and give it a try.
